content/applications/finance/payment_providers/stripe.rst

```diff
@@ -2,132 +2,135 @@
-      **reveal** to display your signing secret.
+      #. Go to the `Webhooks page on Stripe <https://dashboard.stripe.com/webhooks>`_, or log into
+         your Stripe dashboard and go to :menuselection:`Developers --> Webhooks`.
+      #. In the :guilabel:`Hosted endpoints` section, click :guilabel:`Add endpoint`. Then, in the
+         :guilabel:`Endpoint URL` field, enter your Odoo database's URL, followed by
+         `/payment/stripe/webhook`, e.g., `https://yourcompany.odoo.com/payment/stripe/webhook`.
+      #.  Click :guilabel:`Select events` at the bottom of the form, then, in the
+          :guilabel:`Checkout` section, select :guilabel:`checkout.session.completed`, then click
+          :guilabel:`Add events`.
+      #. Click :guilabel:`Add endpoint`, then click :guilabel:`Reveal` and save your
+         :guilabel:`Signing secret` for later.
+      #. In Odoo, :ref:`navigate to the payment provider Stripe
+         <payment_providers/supported_providers>`.
+      #. In the :guilabel:`Credentials` tab, fill the :guilabel:`Webhook Signing Secret` field with
+         the value you previously saved.
+
+      .. note::
+         You can select other events, but they are currently not processed by Odoo.
 
 .. _stripe/local-payment-methods:
 
 Enable local payment methods
 ============================
 
-Local payment methods are payment methods that are only available for certain merchants and
-customers countries and currencies.
+Local payment methods are payment methods that are only available for specific merchants and
+customers' countries and currencies.
 
-Odoo supports the following local payment methods:
+Odoo supports the following local payment methods for Stripe:
 
 - Bancontact
 - EPS
 - giropay
 - iDEAL
 - Przelewy24 (P24)
 
-To enable some of these local payment methods with Stripe, list them as supported payment icons. To
-do so, go to :menuselection:`Payment Providers --> Stripe --> Configuration` and add the desired
-payment methods in the **Supported Payment Icons** field. If the desired payment method is already
-listed, you do not have anything to do.
-
-.. image:: stripe/payments-config.png
-   :alt: Select and add icons of the payment methods you want to enable
+To adapt the list of enabled payment methods, go to the :guilabel:`Configuration` tab and edit the
+list of :guilabel:`Supported Payment Icons`.
 
 .. note::
    - If a payment icon record does not exist in the database and its related local payment method is
      listed above, it is considered enabled with Stripe.
    - If a local payment method is not listed above, it is not supported and cannot be enabled.
 
+Enable Apple Pay
+================
+
+To :ref:`allow customers to use the Apple Pay button <payment_providers/express_checkout>` to pay
+their eCommerce orders, go to the :guilabel:`Configuration` tab, enable :guilabel:`Allow Express
+Checkout`, and click :guilabel:`Enable Apple Pay`.
```

Comment:
Is Google Pay also accepted? If so, I think this header should be changed to reflect that ("Enable Express Checkout" or "Enable Apple Pay and Google Pay") and the rest of the text should include Google Pay as well.